lib/travis/api/v3/renderer/settings.rb

```diff
@@ -0,0 +1,18 @@
+module Travis::API::V3
+  module Renderer::Settings
+    extend self
+
+    AVAILABLE_ATTRIBUTES = [:settings]
+
+    def available_attributes
+      AVAILABLE_ATTRIBUTES 
+    end
+
+    def render(settings, **)
+      {
+        :@type => 'settings'.freeze,
+        :settings => settings.to_h
```

Comment:
Yep, sounds good. I was copying from the V2 response, but top-level sounds better to me. Changing now.